01-why-modern-swift-concurrency/projects/challenge/LittleJohn/Model/LittleJohnModel.swift

```diff
@@ -54,7 +54,7 @@ extension String: Error { }
     for try await line in stream.lines {
       let sortedSymbols = try JSONDecoder()
         .decode([Stock].self, from: Data(line.utf8))
-        .sorted(by: { $0.name < $1.name })
+        .sorted { $0.name < $1.name }
```

Comment:
Hey @iFeli, I'm looking at https://github.com/kodecocodes/mcon/blob/01-why-modern-swift-concurrency/01-why-modern-swift-concurrency/01-why-modern-swift-concurrency.markdown and it seems that this change wasn't backported to the chapter text, could you please do that too?